src/main/java/core/basesyntax/model/ParseLine.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class ParseLine {
```

Comment:
```suggestion
public class ParsedLine {
```
названия классов должны быть существительными